libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestAPI.m

```diff
@@ -273,10 +273,20 @@ - (void)replayRequestIfRequired:(NSData *)data response:(NSURLResponse *)respons
             }
         } else {
             if (!error) {
-                NSDictionary *errorDict = nil;
+                NSMutableDictionary *errorDict = nil;
```

Comment:
Minor: there doesn't appear to be a reason for this to be a mutable dictionary.  Could be a plain old `NSDictionary`.